### PR TITLE
Add Japanese Announcement Blog page

### LINKED ISSUE #17: Add basic content to the website
**Goal**: our use does not lost in this maze of information. I will just add some (existing) content to the overview of the website. 

- [ ] Official Announcement
  - [x] Chinese
  - [ ] English
  - [ ] Japnaese
  - [ ] Italian
- [ ] Guide for general public
  - [x] Chinese
  - [ ] English
  - [ ] Japnaese
  - [ ] Italian

If there are other basic contents that are needed in **this moment**, add to below.

### PR DESCRIPTION
**Pull request by GindaChen**
Fix #17 
Contributor: @GindaChen, @imononoke (Isabella)
Proof Reader: Yudai Yaguchi(@gomyway1216 ), Zoe soen

Future edit of these documents might be needed.

### CODE REVIEW

**Comment by GindaChen** [comment]
@lovepoem I didn't generate the static pages. I think what you meant is that we only commit the changes in README but not commit the static files ourselves.

**Comment by GindaChen** [comment]
@frank-zsy Do we need to generate the static pages ourselves before merge?

File: .gitignore [hunk 1]
```diff
@@ -5,10 +5,10 @@ npm-debug.log
...
-/index.html
-md_json/
-log
+/en-us/	
+/zh-cn/	
+/it-it/	
+/ja-jp/	
+/index.html	
+md_json/	
+log
```
Comment (chenrui333) [inline .gitignore]: 这个看上去像是 editorconfig 的问题。

Comment (GindaChen) [reply .gitignore]: Fix soon

Comment (chenrui333) [reply .gitignore]: 不过 code style 可以后面调整。

**Comment by chenrui333** [comment]
你的 PR title 有点歧义了，好像很多东西都移过来了吧？

**Comment by GindaChen** [comment]
@chenrui333 除了之前的merge以外：
1. 加了Italian / Japanese 的 blog 文件夹
2. 加了Japanese的Announcement Page
3. 加了Japanese的Quickguide

**Comment by GindaChen** [comment]
我可以merge之后再开一个PR

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
我觉得可以合了，看看效果。 😄

**Comment by GindaChen** [comment]
OK 那就merge把

**Comment by chenrui333** [comment]
editorconfig PR, https://github.com/wuhan2020/wuhan2020.github.io/pull/25

**Pull request merged**